apps/gp2-frontend/src/working-groups/__tests__/EditResource.test.tsx

```diff
@@ -0,0 +1,204 @@
+import { gp2 as gp2Model } from '@asap-hub/model';
+import { gp2 as gp2Routing } from '@asap-hub/routing';
+import { gp2 as gp2Fixtures } from '@asap-hub/fixtures';
+import { workingGroups } from '@asap-hub/routing/src/gp2';
+import {
+  render,
+  screen,
+  waitFor,
+  waitForElementToBeRemoved,
+} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { Suspense } from 'react';
+import { MemoryRouter, Route } from 'react-router-dom';
+import { RecoilRoot } from 'recoil';
+import { Auth0Provider, WhenReady } from '../../auth/test-utils';
+import EditResource from '../EditResource';
+import { refreshWorkingGroupState } from '../state';
+import { getWorkingGroup, putWorkingGroupResources } from '../api';
+
+jest.mock('../api');
+
+const renderWorkingGroupDetail = async ({
+  id,
+  userId = '11',
+  route,
+  role = 'Trainee',
+  children,
+}: {
+  id: string;
+  userId?: string;
+  route?: string;
+  role?: gp2Model.UserRole;
+  children: JSX.Element;
+}) => {
```

Comment:
I don't think we should be passing in children as a parameter
It's only for the EditResource modal and should be inlined.

Reply:
Done